Detectors/ITSMFT/MFT/tracking/src/TrackerConfig.cxx

```diff
@@ -42,4 +42,29 @@ void o2::mft::TrackerConfig::initialize(const MFTTrackingParam& trkParam)
   mTrueTrackMCThreshold = trkParam.TrueTrackMCThreshold;
 
   assert(mRPhiBins < constants::index_table::MaxRPhiBins && "Track finder binning overflow");
-}
\ No newline at end of file
+}
+
+//__________________________________________________________________________
+void o2::mft::TrackerConfig::initBins(const BinContainer* mBinContainer)
+{
+  // if it's a non-static method: mutex could be used here
+  // also to protect a BinContainer shared between multiple threads
+
+  // TODO: do nothing at this moment
+  // init the BinContainer
+  // the function initializeFinnder in tracker.cxx is doing this job
+}
+
```

Comment:
this is not needed anymore as all initializations are done in the `initializeFinder`

Reply:
Ok I will remove it